### DISCUSSION TITLE
Any planned support for Gemini / VertexAI APIs in LLM tracing or evaluation?

### DISCUSSION THREAD

**Discussion by tomrom-pomelo** [mlflow/mlflow#12846] [General]
We are looking at mlflow for LLM evaluation right now. We noticed that we can add [custom functions](https://mlflow.org/docs/latest/llms/llm-evaluate/notebooks/question-answering-evaluation.html#Custom-LLM-judged-metric-for-professionalism) for evals, but there didn't seem to be any built-in support for VertexAI / Gemini models in the tracing, evaluation, or deployments. Will these be supported in the future at all?

**Comment by B-Step62** [comment]
@tomrom-pomelo Yes! We are actively expanding the tracing / evaluation support for more model providers. Since there are many providers to be added, the order of support is still TBD. However, please feel free to open a feature request to help getting it proritized!
https://github.com/mlflow/mlflow/issues/new?assignees=&labels=enhancement&projects=&template=feature_request_template.yaml&title=%5BFR%5D
